client/src/components/Cart-dropdown/cart-dropdown-styles.js

```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components";
```

Comment:
cart dropdown that has the ability to toggle when clicking the cart icon in the header